addon/lazy-router.js

```diff
@@ -1,12 +1,12 @@
 import Ember from 'ember';
 
-const get = Ember.get;
+const { get, getOwner } = Ember;
```

Comment:
I don't think so. For apps using older Ember versions, we can continue accessing the container directly.